apps/emqx_authn/src/emqx_authn.erl

```diff
@@ -30,7 +30,16 @@
 
 -include("emqx_authn.hrl").
 
+-dialyzer({nowarn_function, [providers/1]}).
+
 providers() ->
+    providers(emqx_release:edition()).
```

Comment:
It seems that there's a conflict on how to setup EE authenticators: https://github.com/emqx/emqx/pull/11386/files#diff-bc603baaf71ddf57d13fa3467b03a9e50b330e01b1322f3dba295df28093653aR44

Reply:
Ack. I'll follow @lafirest's approach